lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -221,10 +224,10 @@ def _prefetch_scores(self, course_structure, course):
         Returns the prefetched scores for the given student and course.
         """
         if not self._scores_client:
-            scorable_locations = [block_key for block_key in course_structure if possibly_scored(block_key)]
+            scorable_locations = [block_key for block_key in course_structure if possibly_scored(block_key)]  # here
```

Comment:
@nasthagiri, this is the line that causes `3. get student module`, from https://github.com/edx/edx-platform/pull/13262#discussion_r75707531